Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
-COPY . /go/src/github.com/openshift/cluster-machine-appover
-RUN cd /go/src/github.com/openshift/cluster-machine-appover && go build -o machine-approver .
+#
```

Comment:
This less consistent with our default pattern than the previous (which was also not correct)

See https://github.com/openshift/cluster-version-operator/blob/master/Dockerfile for the standard format, you should only have to change the RUN line in the builder and insert your custom steps after the COPY line in the runtime image.

Reply:
:+1: